templates/platforms/xcode/project.yml.hbs

```diff
@@ -81,6 +81,9 @@ targets:
         link: false
       - framework: lib{{snake-case app.name}}.a
         embed: false
+      {{~#each ios-libraries}}
+      - framework: {{this}}
+        embed: false{{/each}}
```

Comment:
Would we want to make this configurable?

Reply:
To expand a bit, we could make the current format set `false` by default, and a format like:
`{framework = "foo", embeded = true}`
also be supported.

Reply:
Yeah, I definitely agree with the YAGNI vibes then.